Userland/Libraries/LibCrypto/Hash/SHA1.cpp

```diff
@@ -7,18 +7,34 @@
 
 #include <AK/Endian.h>
 #include <AK/Memory.h>
+#include <AK/Platform.h>
 #include <AK/Types.h>
 #include <LibCrypto/Hash/SHA1.h>
 
+#if ARCH(I386) || ARCH(X86_64)
+#    include <emmintrin.h> // SSE2 _mm_add_epi32 _mm_loadu_si128 _mm_set_epi32 _mm_set_epi64x _mm_setzero_si128 _mm_shuffle_epi32 _mm_storeu_si128 _mm_xor_si128
+#    include <tmmintrin.h> // SSSE3 _mm_shuffle_epi8
+#    include <smmintrin.h> // SSE4.1 _mm_extract_epi32
+#    include <immintrin.h> // SHA _mm_sha1msg1_epu32 _mm_sha1msg2_epu32 _mm_sha1nexte_epu32 _mm_sha1rnds4_epu32
+#    define sha1_attribute_target_default __attribute__((target("default")))
+#    define sha1_attribute_target_x86 __attribute__((target("sse2,ssse3,sse4.1,sha")))
+#else
+#    define sha1_attribute_target_default
+#    define sha1_attribute_target_x86
```

Comment:
don't need this for non-x86, it's not referenced

Reply:
> don't need this for non-x86, it's not referenced

OK, will fix that.

Reply:
Fixed.